distribution/config/nginx.conf

```diff
@@ -122,7 +122,7 @@ http {
         }
 
         location / {
-            if ( -f /evergreen/booting.txt ) {
+            if ( -f /evergreen/data/booting.txt ) {
```

Comment:
Typically shouldn't it be left above on purpose? I mean: we should put things in `$EVERGREEN_DATA` we care about to keep for long. AFAICT, we do not care about the booting.txt file staying over a `docker stop` and new pull do we?

Reply:
I thought about that too. This keeps things a bit simpler in the client in that it will not _write_ anything outside of `EVERGREEN_DATA` which is IMHO a good thing.